paradox-nakadi-consumer-core/src/main/java/de/zalando/paradox/nakadi/consumer/core/http/handlers/EventUtils.java

```diff
@@ -40,18 +45,17 @@ private EventUtils() { }
 
                 final ArrayNode arrayNode = (ArrayNode) eventsNode;
                 arrayNode.elements().forEachRemaining(element -> {
-                    if (!element.isNull()) {
```

Comment:
Probably yes. It's now in checkEventType(), where it makes sense, since it is a different method.